lib/tweet_deletion/tester.rb

```diff
@@ -106,7 +106,7 @@ def is_rt()
     end
 
     def links_to(match)
-      match = Regexp("^#{Regexp.quote(match)}$") if match.kind_of? String
+      match = Regexp.quote(match) if match.kind_of? String
```

Comment:
This is a change in behaviour. With the previous code, you can pass a link and it will check for an exact match. With this change, it will also match a link with additional parameters. I don't think this would be expected by the user.

`match = Regexp.new("^#{Regexp.quote(match)}$") if match.kind_of? String` ?

Reply:
Hmm, indeed what you propose fixes it. That being said now I realize that I misunderstood that the intent of links_to was for exact links. I was using it to check if the toot was linking anywhere in my website, exact matches won't cut it.

Should I add new API for my case? Not sure how to name such a new function though.

Reply:
It should already be covered. Sending a string was expected to be an exact match but you can also directly pass a regexp like `links_to.match( %r{^https?://yourdomain.com\} )`

Reply:
Oh right, stupid me, indeed works with links_to(%r{...}).

I'll upload a revised fix for the regexp code shortly.